compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -16,6 +16,7 @@ pub enum InterpreterError {
     ArgumentCountMismatch { expected: usize, actual: usize, location: Location },
     TypeMismatch { expected: Type, value: Value, location: Location },
     NonComptimeVarReferenced { name: String, location: Location },
```

Comment:
Is this used anymore? It seems that `VariableNotInScope` replaces all of its uses.

Reply:
I've just re-added the check for it now so that we still issue this error if the variable is defined but not in scope